src/index.tsx

```diff
@@ -3,10 +3,11 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
```

Comment:
Is app still being used then? And if not, will it be used again in the future?

Reply:
Yes the reason is because we just want to display the BlogPost component

Reply:
we will switch back to the app in the next module